python/iceberg/api/transforms/dates.py

```diff
@@ -17,7 +17,7 @@
 
 import datetime
 
-from .projection_util import ProjectionUtil
+from . import projection_util
```

Comment:
nit: we could do `from .projection_util import truncate_integer`

Not a big deal though...

Reply:
SG